### DISCUSSION TITLE
package-lock version

### DISCUSSION THREAD

**Discussion by krau5** [npm/feedback#268] [CLI] [answered]
Hey everyone! I've got some problems while installing needed packages in my repo.
Which problems? Recently, I updated my npm to new version but my repo doesn't work with second version of package-lock.
In that case, I think that'd be great if you made it possible to choose needed package-lock version while installing.

**Comment by MylesBorins** [comment, marked as answer]
hey @Krausso are you adding new dependencies or just installing the existing ones? If you use `npm ci` rather than `npm install` npm 7 will install your packages as per the v1 lockfile and not update anything.

If you are installing new packages in the repo you should use npm 6 while adding the new dependencies if you want to continue to use the v1 lockfile as npm 7 only support the v2 lockfile when updating your package.json and package-lock.json

**Reply by krau5** [reply]
I use `npm i`, so package-lock version often changes to v2 instead of v1.
Yeah, I've already downgraded to npm v6. However, I really want to use the last one.

Nah, anyway, I think specified feature was not so bad to include.
Have a nice day:)
